content/using_ec2_spot_instances_with_eks/scaling/test_ca.md

```diff
@@ -153,14 +157,10 @@ It is a recommended to use **[capacity-optimized](https://aws.amazon.com/blogs/c
 ### Optional Exercises
 
 {{% notice warning %}}
-Some of this exercises will take time for Cluster Autoscaler to scale up and down. If you are running this
-workshop at a AWS event or with limited time, we recommend to come back to this section once you have 
-completed the workshop, and before getting into the **cleanup** section.
+Some of this exercises will take time for Cluster Autoscaler to scale up and down. If you are running this workshop at a AWS event or with limited time, we recommend to come back to this section once you have completed the workshop, and before getting into the **cleanup** section.
 {{% /notice %}}
 
- * At the moment AWS auto-scaling groups backing up the nodegroups are setup to use the [capacity-optimized allocation strategy](https://docs.aws.amazon.com/en_pv/autoscaling/ec2/userguide/asg-purchase-options.html#asg-allocation-strategies). What do you think is the trade-off when you switch to [lowest price](https://docs.aws.amazon.com/en_pv/autoscaling/ec2/userguide/asg-purchase-options.html#asg-allocation-strategies) allocation strategy ? 
-
- * What will happen when modifying Cluster Autoscaler **expander** configuration from **random**  to **least-waste**. What happens when we increase the replicas back to 13 ? What happens if we increase the number of replicas to 20? Can you predict which group of node will be expanded in each case: (a) 4vCPUs 16GB RAM (b) 8vCPUs 32GB RAM? What's Cluster Autoscaler log looking like in this case?
+ * What will happen when modifying Cluster Autoscaler **expander** configuration from **random**  to **least-waste**. What happens when we increase the replicas back to 23 ? What happens if we increase the number of replicas to 30? Can you predict which group of node will be expanded in each case: (a) 4vCPUs 16GB RAM (b) 8vCPUs 32GB RAM? What's Cluster Autoscaler log looking like in this case?
```

Comment:
to 23 ?

remove space before question mark

Reply:
As Ran pointed out remove the space between 23 and ?